lib/commands/general.js

```diff
@@ -47,6 +47,22 @@ commands.getWindowSize = async function () {
   return await this.uiautomator2.jwproxy.command('/window/current/size', 'GET', {});
 };
 
+commands.execute = async function (script, args) {
+  if (script.match(/^mobile\:/)) {
+    script = script.replace(/^mobile\:/, '').trim();
+    if (script === 'scroll' && !this.isWebContext()) {
+      let elementId = args[0].element;
+      return await this.uiautomator2.jwproxy.command(`/appium/element/${elementId}/scroll_to`, 'POST', {});
+    } else if (script === 'viewportScreenshot' && !this.isWebContext()) {
+      return await this.getViewportScreenshot();
+    } else {
```

Comment:
these two `else` blocks are not needed. this.executeCommand after `if` block is enough, since the internal condition anyway perform _return_